src/components/TimeRangePicker/TimeRangePicker.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
```

Comment:
also create `index.js` file with import `TimeRangePicker` and export `TimeRangePicker`
It need for short import in other files
See examples in other components